### PR TITLE
minor fix in test_create_torrent v2_only

### PR DESCRIPTION
**Pull request by aldenml**

### CODE REVIEW

File: test/test_create_torrent.cpp [hunk 1]
```diff
@@ -236,13 +236,13 @@ TORRENT_TEST(v2_only)
...
 	TEST_EQUAL(info.files().file_name(0_file), "A");
-	TEST_EQUAL(info.files().pad_file_at(1_file), true);
+	TEST_CHECK(info.files().pad_file_at(1_file));
 	TEST_EQUAL(info.files().file_name(2_file), "B");
 	TEST_EQUAL(info.name(), "test");
 
 	lt::create_torrent t2(info);
 	std::vector<char> buffer2;
-	lt::bencode(std::back_inserter(buffer2), t.generate());
+	lt::bencode(std::back_inserter(buffer2), t2.generate());
```
Comment (arvidn) [inline test/test_create_torrent.cpp:245]: ouch, this is a bad typo. It should probably go into `RC_2_0` as well

**Comment by arvidn** [comment]
would it be a lot of work to rebase onto `RC_2_0`?

**Comment by aldenml** [comment]
no a problem

**Comment by aldenml** [comment]
rebased onto `RC_2_0`

**Pull request merged**